### DISCUSSION TITLE
Alternative of OAuth: Removed OAuth Support in HTTP task

### DISCUSSION THREAD

**Discussion by pankajthamkeappdirect** [Netflix/conductor#2603] [Q&A]
Hi Team, in the change log it is mentioned that **Removed OAuth Support in HTTP task** Most of the workflows that we have use HTTP task with OAUTH, Is there any alternative ?

**Comment by jxu-nflx** [comment]
Hello @pankajthamkeappdirect we are currently not supporting OAuth in Http task, and unfortunately there is no alternative solution for that. We can add that as feature request into our backlog, but might not be able to get to it soon. You are more than welcome to add that support on your own and create a PR for it!

**Reply by jxu-nflx** [reply]
https://github.com/Netflix/conductor/issues/2606

**Comment by jkaipa** [comment]
> Hi Team, in the change log it is mentioned that **Removed OAuth Support in HTTP task** Most of the workflows that we have use HTTP task with OAUTH, Is there any alternative ?

Was there a reason that this support was removed?
